README.md

```diff
@@ -1 +1,31 @@
-# googleapi1
\ No newline at end of file
+Installation & Usage
+
+Installation
+
+Clone or download the repo.
+Open terminal and navigate to Google folder.
+Run npm install to install dependencies.
+
+Usage
+Navigate to server.
+Run nodemon app.js to launch the server.
+Navigate to index.html in client then open with live server.
+
+Tests
+Run npm test to launch test suite(in progress).
```

Comment:
Had to change directory to /client to then install dependencies and then run npm test. Not specified in README.